dagster_polars/io_managers/base.py

```diff
@@ -146,20 +156,23 @@ class BasePolarsUPathIOManager(ConfigurableIOManager, UPathIOManager):
     """
 
     base_dir: Optional[str] = Field(default=None, description="Base directory for storing files.")
-
-    _base_path: "UPath" = PrivateAttr()
+    cloud_storage_options: Optional[Mapping[str, Any]] = Field(
```

Comment:
Can we remove the `Optional` and just have `{}` as the default? 
Also, isn't it better to call it just `storage_options`?

Reply:
You can't call it storage_options because the storage_options is a property on the UPathIOmanager, so it will throw a pydanticvalidation error, so that's why we need to call it something different.

Why do you want to remove the optional? It is optional, all the other libraries that allow storage_options hint it as optional.

Reply:
Removing the `Optional` would just make life a little easier when using this field. 
Might also make the config more readable in Dagster's UI.
There is no chance for `storage_options` to actually be `None`, as you are doing `self.cloud_storage_options or {}` anyway, so I think it makes sense to just have `{}` as default value.

Reply:
I don't agree :) The storage_options can be None because either you are access a local object store or you have set the strorage_options as environment variables, which polars and delta-rs will pick up through the object storage implementation.

Say that it's not optional therefore is technically incorrect and can be confusing imho